src/App.tsx

```diff
@@ -1,61 +1,111 @@
 import './App.scss';
+import users from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const initialTodos: Todo[] = todosFromServer;
+
+function getNewTodoId(todos: Todo[]): number {
+  const maxId = Math.max(...todos.map(todo => todo.id), 0);
+
+  return maxId + 1;
+}
+
+export function getUserById(userId: number): User | null {
+  return users.find(user => user.id === userId) || null;
+}
```

Comment:
The `getUserById` function should not be defined in this file. As per the previous review, data preparation should be done in the App component, and necessary data should be passed down to child components. Consider moving this logic to the App component and passing the required user data as props to the `TodoInfo` component.